docs/index.html

```diff
@@ -1026,8 +1026,177 @@ <h3 id="col-bleed-y">.col-bleed-y <a href="#col-bleed-y">&#35;</a></h3>
 
             <section class="container">
                 <h2 id="visibility-helpers">Visibility helpers <a href="#visibility-helpers">&#35;</a></h2>
-                <p>You can use some generic visibility helpers that can be applied to any element, not just grid elements. resize the viewport to see them in action. The breakpoints can be changed if you're building the css yourself by building the SCSS files.</p>
-
+                <p>You can use some generic visibility helpers that can be applied to any element, not just grid elements. 
+                    <br>The <code>.hidden-*-up</code> classes hide the element when the viewport is at the given breakpoint or wider, 
+                    the <code>.hidden-*-down</code> classes hide the element when the viewport is at the given breakpoint or smaller and <code>.hidden-*</code>
+                    classes hide the element when the viewport is at the given breakpoint.
+                    <br>The breakpoints can be changed if you're building the css yourself by building the SCSS files.</p>
```

Comment:
Would prefer new `<p>` here and use similar language as used elsewhere on the page:

"It is possible to change these breakpoints by modifying the included variables.scss file and building a new css file."

Reply:
Ok, understandable.